NitroxInstallerActions/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="dnlib" version="1.6.1" targetFramework="net461" />
```

Comment:
Should we add a git submodule for dnlib? (https://github.com/0xd4d/dnlib)

Reply:
That is up to you. If you want I will add it as a sub module.